php/libraries/OutputWrapper.class.inc

```diff
@@ -39,15 +38,21 @@ class OutputWrapper
     /**
      * Create a new command line helper.
      *
-     * @param ?SPLFileInfo $logDirectory The directory that will contain log
-     *                                   files written by this script.
+     * @param ?SPLFileInfo $logDirectory The full path to the directory that
+     *                                   will contain log files written by this
+     *                                   script.
      *
      * @return void
      */
     public function __construct(?SPLFileInfo $logDirectory = null)
     {
+        $config = \NDB_Factory::singleton()->config();
         $logDir = new SPLFileInfo(
-            $logDirectory ?? \NDB_Factory::singleton()->config()->getSetting('log')
+            $logDirectory ??
+            \Utility::pathJoin(
+                $config->getSetting('base'),
```

Comment:
I don't think this is going to be correct if the log setting is an absolute path.

Reply:
Fortunately this is the only place in the codebase where the setting is being used

<img width="714" alt="Screen Shot 2020-07-07 at 16 45 42" src="https://user-images.githubusercontent.com/4022790/86841020-68c80d80-c071-11ea-82c2-79e856c1af91.png">

Reply:
okay, but it's still going to be wrong if the log directory is set to an absolute path (ie. `/var/log/loris`)..

Reply:
So what's the solution?

The same can be said of the detect_conflicts script. The status quo right now is that, to the extent `log` is used, it's a partial path stored relative to `base` so this is fine.

It's possible to change it to instead change it to an absolute path: we can change the SQL to be of a `web_path` type which would cause the Config module to validate it when submitting. That would require updating `detect_conflicts` as well. It's probably not a bad idea for a separate pull request. 

But right now this is just using a config setting as it's expected to be used so I'm not sure what you want.

Reply:
Why can't you just check if it starts with a `/` and not join if so? You said this is the only place it's being used..

Reply:
Other than the tools/detect_conflicts.php script, yes

Reply:
done